ee/tabby-ui/components/ui/codeblock.tsx

```diff
@@ -145,7 +145,7 @@ const CodeBlock: FC<CodeBlockProps> = memo(
                     variant="ghost"
                     size="icon"
                     className="text-xs hover:bg-[#3C382F] hover:text-[#F4F4F5] focus-visible:ring-1 focus-visible:ring-slate-700 focus-visible:ring-offset-0"
-                    onClick={onApplyInEditor.bind(null, value, undefined)}
+                    onClick={() => onApplyInEditor(value, undefined)}
```

Comment:
Using `bind` may cause bugs on some platforms. A lambda should be good for both readability and compatibility.